Automattic/wp-calypso
# Add the all sites domain management paths

#### Changes proposed in this Pull Request

* In order to keep the context when managing your domain via the all sites view we need to duplicate every domain management route.

Don't panic - the PR looks bigger than it actually is.

#### Testing instructions

From an all sites domains view (/domains/manage) you should be able to navigate all the domain management screens without this changing the left navigation bar. In more details this PR handles:

* Site redirect domain status page and it's back button
* Incoming transfer domain status page and the back button
* Dns settings for mapped domain and the back button
* Name servers and dns for registered domains and the back button
* Update your contact information, Edit contact info and Manage consent and all back buttons
* Connect your domain for mapped domain with Domain Connect
* Review your domain security
* Transfer domain and all screens in that section
* Change site address

The thing that is missing in this PR is the email routes since they're a bit different than the rest.
